### DISCUSSION TITLE
admin url not found after installation

### DISCUSSION THREAD

**Discussion by infiniteroles** [Laravel-Backpack/community-forum#1110] [Q&A (Help)] [answered]
I have performed a clean installation of Laravel 11 and initially, everything seems fine. https://rnbservice.infiniteroles.com/

After that, I installed Backpack and created my first CRUD, but both the admin URL and the specific CRUD URL show up as not found 404. It’s been a while since my last Backpack installation, but I swear I’ve done all the steps correctly. Does it ring a bell if I might be missing something?

Creating Model app/Models/User.php ……………………………………………………………….. ALREADY EXISTED
Adding CrudTrait to the Model ……………………………………………………………………. ALREADY EXISTED
Creating Controller app/Http/Controllers/Admin/UserCrudController.php ………………………………… ALREADY EXISTED
Creating Request app/Http/Requests/UserRequest.php …………………………………………………. ALREADY EXISTED
Adding route to /var/www/html/laravel/routes/backpack/custom.php …………………………………….. ALREADY EXISTED
Adding menu entry to resources/views/vendor/backpack/ui/inc/menu_items.blade.php ………………………. ALREADY EXISTED

Done! Go to https://rnbservice.infiniteroles.com/admin/user to see the CRUD in action.

**Comment by jcastroa87** [comment]
Hello @infiniteroles 

Is you APP_URL correctly set in .env? is this https?

This look as a server config issue than a Backpack problem.

Please let me know what is your server config, is VPS, local? Docker? Backpack version.

Thanks.

Cheers.

**Reply by infiniteroles** [reply]
IT´s an VPS, with las backpack version.
mi .env file have tis config:
APP_NAME=Laravel
APP_ENV=local
APP_KEY=******
APP_DEBUG=true
APP_TIMEZONE=UTC
APP_URL=https://rnbservice.infiniteroles.com

APP_LOCALE=en
APP_FALLBACK_LOCALE=en
APP_FAKER_LOCALE=en_US

APP_MAINTENANCE_DRIVER=file
# APP_MAINTENANCE_STORE=database

BCRYPT_ROUNDS=12


I reinstalled backpack but get the same error:
sudo php artisan backpack:install


   STEP 1  Installing Backpack CRUD.

  Publishing configs, views, routes and assets ........................................................................... DONE
  Creating users table ................................................................................................... DONE
  Creating CheckIfAdmin middleware ....................................................................................... DONE
  Installing Generators .................................................................................................. DONE
  Installing Basset ...................................................................................................... DONE

   STEP 2  Installing a Theme.

  │ Choose your admin UI, depending on your project and preferences.

  Tabler (default) was already installed



   STEP 3  Creating an admin.

  │ Quickly jump in your admin panel, using the email & password you choose here.
  │ Currently there is 1 user in the users table.
  │ Skipping creating an admin user.


   STEP 4  Installing premium Backpack add-ons:

  │ Add tons of features and functionality to your admin panel, using our paid add-ons.
  │ For more information, payment and access please visit https://backpackforlaravel.com/pricing

  1 Backpack Pro ................................................................................................ NOT INSTALLED
    Adds 5 operations, 10 filters, 28 fields, 6 columns, charts

  2 DevTools .................................................................................................... NOT INSTALLED
    Helps generate models, migrations, operations and CRUDs

  3 Editable Columns ............................................................................................ NOT INSTALLED
    Allow your admins to make small changes from the table view



   INFO  Checking Backpack Basset installation.

  Initializing basset check ................................................................................................................... DONE
  Checking cache storage ...................................................................................................................... DONE
  Fetching a basset ........................................................................................................................... DONE


   DONE  Backpack installation complete.

  │ Go to https://rnbservice.infiniteroles.com/admin/ to access your new admin panel.
  │ You may need to run php artisan serve to serve your Laravel project.

**Comment by jcastroa87** [comment]
Can you share the Apache configuration and if there is something in Apache logs.

Cheers.

**Reply by infiniteroles** [reply]
this is my virtual host config:

<IfModule mod_ssl.c>
<VirtualHost *:443>
ServerAdmin webmaster@localhost
ServerName rnbservice.infiniteroles.com
ServerAlias www.rnbservice.infiniteroles.com
DocumentRoot /var/www/html/laravel/public
ErrorLog ${APACHE_LOG_DIR}/error.log
CustomLog ${APACHE_LOG_DIR}/access.log combined

SSLCertificateFile /etc/letsencrypt/live/rnbservice.infiniteroles.com/fullchain.pem
SSLCertificateKeyFile /etc/letsencrypt/live/rnbservice.infiniteroles.com/privkey.pem
Include /etc/letsencrypt/options-ssl-apache.conf
</VirtualHost>
</IfModule>

this is the info i see in access log:
45.156.128.49 - - [08/Aug/2024:18:37:25 +0200] "GET /OA_HTML/AppsLocalLogin.jsp HTTP/1.1" 404 492 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/60.0.3112.113 Safari/537.36"
149.50.103.48 - - [08/Aug/2024:18:42:38 +0200] "GET / HTTP/1.1" 200 35201 "-" "-"
90.165.253.233 - - [08/Aug/2024:18:46:11 +0200] "GET /admin/ HTTP/1.1" 404 4403 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/127.0.0.0 Safari/537.36"
90.165.253.233 - - [08/Aug/2024:18:46:12 +0200] "GET /favicon.ico HTTP/1.1" 304 324 "https://rnbservice.infiniteroles.com/admin/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/127.0.0.>
90.165.253.233 - - [08/Aug/2024:18:46:12 +0200] "GET /admin/ HTTP/1.1" 404 585 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/127.0.0.0 Safari/537.36"
90.165.253.233 - - [08/Aug/2024:18:46:13 +0200] "GET /admin/ HTTP/1.1" 404 4403 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/127.0.0.0 Safari/537.36"
90.165.253.233 - - [08/Aug/2024:18:58:10 +0200] "GET /admin/ HTTP/1.1" 404 4403 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/127.0.0.0 Safari/537.36"
90.165.253.233 - - [08/Aug/2024:18:58:11 +0200] "GET /admin/ HTTP/1.1" 404 585 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/127.0.0.0 Safari/537.36"
90.165.253.233 - - [08/Aug/2024:18:58:13 +0200] "GET /admin/ HTTP/1.1" 404 585 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/127.0.0.0 Safari/537.36"
90.165.253.233 - - [08/Aug/2024:18:58:16 +0200] "GET /admin/ HTTP/1.1" 404 4403 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/127.0.0.0 Safari/537.36"
90.165.253.233 - - [08/Aug/2024:18:58:46 +0200] "GET /admin/ HTTP/1.1" 404 4403 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/127.0.0.0 Safari/537.36"
90.165.253.233 - - [08/Aug/2024:18:58:48 +0200] "GET / HTTP/1.1" 200 10468 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/127.0.0.0 Safari/537.36"
90.165.253.233 - - [08/Aug/2024:18:58:59 +0200] "GET /admin HTTP/1.1" 404 4403 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/127.0.0.0 Safari/537.36"
90.165.253.233 - - [08/Aug/2024:18:59:00 +0200] "GET /admin HTTP/1.1" 404 909 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/127.0.0.0 Safari/537.36"


I don´t se enything related in error log:
[Thu Aug 08 18:57:14.551087 2024] [mpm_prefork:notice] [pid 674] AH00170: caught SIGWINCH, shutting down gracefully
[Thu Aug 08 18:57:15.679272 2024] [mpm_prefork:notice] [pid 7809] AH00163: Apache/2.4.52 (Ubuntu) OpenSSL/3.0.2 configured -- resuming normal operations
[Thu Aug 08 18:57:15.679317 2024] [core:notice] [pid 7809] AH00094: Command line: '/usr/sbin/apache2'

Thanks so mmuch for your help

**Comment by infiniteroles** [comment, marked as answer]
I have reinstalled LAravel from scratch and now it´s working fine
